docs/source/traits.rst

```diff
@@ -0,0 +1,69 @@
+===========
+Trait Types
+===========
+
+Pyface defines a number of custom Trait types that represent quantities and
+objects that are useful in the context of graphical user interfaces.
+
+Colors
+======
+
+When working with user interfaces, it is common to want to be able to specify
+the color to use in part of the UI.  Each toolkit usually has its own way of
+representing colors, and so the ability to specify a color in a
+toolkit-independent way that can be converted to a toolkit-specific
+representation is important.  This is particularly so when you want to allow
```

Comment:
```suggestion
representation is important.  This is particularly important when you want to allow
```

Reply:
I think I prefer the "so" - it references pack to the "important" in the previous sentence without repeating the word.